### ISSUE TITLE
Incremental Build in Visual Studio 2019 and WPF with .NET Core 3.0 broken

### ISSUE THREAD

**Issue by TFTomSun** [dotnet/wpf#1981]
- Create a library with the following project file:

```xml

<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">

  <PropertyGroup>
    <TargetFramework>netcoreapp3.0</TargetFramework>
    <UseWPF>true</UseWPF>
  </PropertyGroup>

</Project>


```

- To keep it simple, leave the library empty

-> The delta build in Visual Studio is broken. Normally Visual Studio would not build the project on the second build, if no changes have been performed. But WPF libraries are always build again, which also causes dependent projects to be rebuild. WPF apps don't have that issue.

Unfortunately the diagnostic build output does not tell any reason why the project is build again.

**Comment by weltkante** [comment]
#460 looks related but the scenario differs in the details - could be the same underlying issue though. Unfortunately #460 has been postponed to 'Future' (i.e. not planned for 3.1 or 5.0) which sounds weird considering its a regression in the build experience?

**Comment by ryalanms** [comment]
This is likely fixed by @MichaeIDietrich: https://github.com/dotnet/wpf/pull/4053.

**Comment by ryalanms** [comment]
Fixed by @MichaeIDietrich: #4053.  Please reactivate if you see any new incremental build issues.
